cyences_app_for_splunk/bin/device_inventory_v2_util.py

```diff
@@ -13,34 +12,100 @@
 Dev Details
 * It now stores everything inside a Python pickle file instead of a lookup
 * Most of the processing now happens inside the Python script.
-'''
+"""
 
-import os
-import pickle
 import uuid
 import copy
+import json
+import cs_utils
+
+from splunk import rest
 
 
-MAX_TIME_EPOCH = 2147483647   # Tue Jan 19 2038 03:14:07
+MAX_TIME_EPOCH = 2147483647  # Tue Jan 19 2038 03:14:07
+
+CY_HOSTNAME_POSTFIXES_MACRO = "cs_device_inventory_hostname_postfixes"
```

Comment:
I don't think this is required here as we are passing it as class argument.

Reply:
We are passing the values of the hostname_postfixes from this macro.

Reply:
Yes, but we are getting it as an argument. It is defined in two python files.

Reply:
It's my bad. removed unused variable.